stacks/billing-stack.js

```diff
@@ -179,5 +179,36 @@ export function BillingStack ({ stack, app }) {
     CustomDomain: customDomain ? `https://${customDomain.domainName}` : 'Set BILLING_HOSTED_ZONE in env to deploy to a custom domain'
   })
 
+  // Lambda that handles egress traffic tracking
+  const egressTrafficQueueHandler = new Function(stack, 'egress-traffic-queue-handler', {
+    permissions: [customerTable],
+    handler: 'billing/functions/egress-traffic-queue.handler',
+    timeout: '15 minutes',
+    bind: [stripeSecretKey],
+    environment: {
+      AWS_REGION: stack.region,
+      CUSTOMER_TABLE_NAME: customerTable.tableName,
+      // TODO (fforbeck): make this a config based on the env: local, staging, prod
```

Comment:
I'm not sure how to make this a config. Any tips?

Reply:
actually, thinking this through, I'm not so sure this needs to be different per-env - staging will act against the Stripe test environment already and we probably want the meter event name to be the same there as in prod, I think? could definitely be convinced, but I think it might make sense just to pull this out to a `const` and pick a good name for it - `gateway-egress-traffic` seems reasonable to me?

Reply:
That makes sense to me. The `gateway-egress-traffic` is the one I was using already, I will remove the `test-` prefix. Thank you!